Maquetado/scss/details.scss

```diff
@@ -4,15 +4,44 @@
   flex-wrap: wrap;
 }
 
+@mixin shadow {
+  box-shadow: 4px 4px 10px 0 $black-transparent;
+}
+
+@mixin texto($size,$weight,$height) {
+  font-size: $size;
+  font-weight: $weight;
+  line-height: $height;
+}
+
+.background {
+  background-color: $wild-sand;
+  height: 100%;
+  width: 100%;
+}
+
+.back-button {
+  font-size: 20px;
+  line-height: 24px;
+  margin: 50px 30px;
+}
+
+.back-button::before {
+  content: url('assets/back-arrow-20.png');
```

Comment:
In this case this work it because the image is small, but you should have present that the correct manner is using `background-image` property